autogluon/utils/tabular/data/cleaner.py

```diff
@@ -61,10 +62,10 @@ def get_valid_classes(X, label, threshold):
         sum_after = sum(class_counts_valid)
         percent = sum_after / sum_prior
         if len(valid_classes) < len(class_counts):
-            logger.log(25, 'Warning: Some classes in data were not valid. AutoGluon will only keep '+str(len(valid_classes))+
-                   ' out of '+str(len(class_counts))+ ' classes')
+            logger.log(25, 'Warning: Some classes in the training set have fewer than %s examples. AutoGluon will only keep %s out of %s classes for training. '
+                           'To keep more classes, increase data size or reduce label_count_threshold.' % (threshold, len(valid_classes), len(class_counts)))
```

Comment:
"classes for training"
-> 
"classes for training and will not try to predict the rare classes"

"increase data size" 
-> 
"increase the number of datapoints from these rare classes in the training data"

Reply:
Updated